docker-compose.yaml

```diff
@@ -0,0 +1,66 @@
+version: '3.8'
+
+services:
+  postgres:
+    restart: always
+    container_name: aana_summarize_video_db
+    image: postgres
+    command: postgres -c 'max_connections=1000'
+    healthcheck:
+      test: /usr/bin/pg_isready
+      timeout: 45s
+      interval: 10s
+      retries: 10
+    ports:
+      - '15430:15430'
+    expose:
+      - 15430
+    environment:
+      PGPASSWORD: '${POSTGRES_PASSWORD:-Yf?5nX39}'
+      PGUSER: '${POSTGRES_USER:-aana_db_user}'
+      PGDATABASE: '${POSTGRES_DB:-aana_db}'
+      POSTGRES_PASSWORD: '${POSTGRES_PASSWORD:-Yf?5nX39}'
+      POSTGRES_USER: '${POSTGRES_USER:-aana_db_user}'
+      POSTGRES_DB: '${POSTGRES_DB:-aana_db}'
+      PGPORT: '15430'
+      PGDATA: '/pgdata'
+    volumes:
+      - pg_data:/pgdata
+    
+  aana_summarize_video_app:
+    restart: always
+    container_name: aana_summarize_video_app
+    depends_on:
+      postgres:
+        condition: service_healthy
+    ports:
+      - 8000:8000   # request server
+    expose:
+      - '8000'
+    build:
+      context: .
+      dockerfile: Dockerfile
+    deploy:
+      resources:
+        reservations:
+          devices:
+              - capabilities: ["gpu"]
+    environment:
+      CUDA_VISIBLE_DEVICES: '${CUDA_VISIBLE_DEVICES}'
+      HF_HUB_ENABLE_HF_TRANSFER: '${HF_HUB_ENABLE_HF_TRANSFER:-1}'
+      HF_TOKEN: '${HF_TOKEN}'
+      HF_DATASETS_CACHE: /root/.cache/huggingface
+      NUM_WORKERS: '${NUM_WORKERS:-2}'
+      TMP_DATA_DIR: /tmp/aana_data
+      DB_CONFIG: '{"datastore_type":"postgresql","datastore_config":{"host":"postgres","port":"15430","user":"${POSTGRES_USER:-aana_db_user}","password":"${POSTGRES_PASSWORD:-Yf?5nX39}","database":"${POSTGRES_DB:-aana_db}"}}'
+    volumes:
+      - app_data:/tmp/aana
```

Comment:
It still points to the wrong directory. It should be `/tmp/aana_data` like you set it above.

Reply:
Fixed.